pkg/client/proxy.go

```diff
@@ -154,3 +172,9 @@ func (c *ProxyClient) ClientVersionGet() (version emeta.VersionOutput) {
 	logrus.Trace("Getting client version")
 	return emeta.GetVersion()
 }
+
+func (c *ProxyClient) CheckConnection() error {
+	req := &healthpb.HealthCheckRequest{}
+	_, err := c.health.Check(getContextWithGRPCTimeout(c.ctx), req)
+	return err
+}
```

Comment:
This seems like a good solution to me. We already have the health check endpoint available in the server, so we may as well make use of it (instead of adding a new API).